package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Moramia website",
   "scripts": {
-    "prepare-public": "node prepare.js"
+    "build": "webpack & node prepare.js"
```

Comment:
`&` makes both process to be parallel, use `&& `to make sure the first process finished before executing the next one.